projects/kit/src/lib/masks/number/utils/tests/stringify-number.spec.ts

```diff
@@ -0,0 +1,386 @@
+import {describe, expect, it} from '@jest/globals';
+
+import {
+    CHAR_EM_DASH,
+    CHAR_EN_DASH,
+    CHAR_HYPHEN,
+    CHAR_JP_HYPHEN,
+    CHAR_MINUS,
+} from '../../../../constants';
+import {maskitoStringifyNumber} from '../stringify-number';
+
+describe('maskitoStringifyNumber', () => {
```

Comment:
Add test when thousand separator is the point and the first argument is a number with decimal part.

Read this comment:
* https://github.com/taiga-family/maskito/issues/1966#issuecomment-2664914754

Reply:
solved